### DISCUSSION TITLE
Install on macOS without Go

### DISCUSSION THREAD

**Discussion by dpokluda** [JanDeDobbeleer/oh-my-posh#3157] [Troubleshoot]
Because of our company IT policy, I can't get GO toolset downloaded/installed on my macOS machine. Is there an alternative way (similar to manual/DEB installation for Ubuntu) to install oh-my-posh on macOS?

Thanks.

**Comment by JanDeDobbeleer** [comment]
@dpokluda you can always download the binary from releases here.

**Reply by dpokluda** [reply]
Thank you @JanDeDobbeleer and I apologize for the following question - which binary (release) do I take for M1 MBP?

**Reply by JanDeDobbeleer** [reply]
@dpokluda the [arm](https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v12.23.0/posh-darwin-arm64) version.

**Reply by dpokluda** [reply]
Thank you Jan.
